submissions/Levi123/domjs/script.js

```diff
@@ -0,0 +1,64 @@
+    },
+    {
+        id:4,
+        series: '4 series',
+        images: 'https://www.bmw.ua/content/bmw/marketB4R1/bmw_ua/uk_UA/all-models/4-series/coupe/2020/bmw-4-series-coupe-highlights/jcr:content/par/mosaicgallery/items/mosaicgalleryitem_2045026459/image.transform/mosaic485/image.1597922414251.jpg',
+        information: 'Безкомпромісний нонконформіст: абсолютно новий BMW 4 серії Coupé рішуче та елегантно виходить за рамки всіх сучасних правил. Незалежний дизайн і класичний силует купе підкреслюють його прогресивний характер. Спортивна легкість плавно переходить на дорогу, де вражаюча маневреність і привабливі характеристики сприяють підвищенню рівня адреналіну.',
+    },
+    {
+        id:5,
+        series: '5 series',
+        images: 'https://www.bmw.ua/content/bmw/marketB4R1/bmw_ua/uk_UA/all-models/5-series/sedan/2020/bmw-5-series-sedan-highlights/jcr:content/par/mosaicgallery_d34b/items/mosaicgalleryitem_97/image.transform/mosaic963/image.1597323989179.jpg',
+        information: 'BMW 5 серії Седан — втілення спортивного бізнес-седана. З першого погляду він демонструє стильний атлетизм, який особливо виражається у динамічному силуеті та лаконічному дизайні. Прості поверхні та чіткі контури надають елегантному зовнішньому вигляду сучасну та технологічну атмосферу. Елегантний і функціональний інтерєр доповнює інноваційні амбіції автомобіля технологіями майбутнього та спортивними особливостями. зкомпромісний нонконформіст: абсолютно новий BMW 4 серії Coupé рішуче та елегантно виходить за рамки всіх сучасних правил. Незалежний дизайн і класичний силует купе підкреслюють його прогресивний характер. Спортивна легкість плавно переходить на дорогу, де вражаюча маневреність і привабливі характеристики сприяють підвищенню рівня адреналіну.',
+    },
+]
+
+
+const button = document.querySelector('.about-product__navigation')
+const mainInfoBlock = document.querySelector('.about-product__info-block');
+const mainTitle = document.createElement('h1');
+const mainImage = document.createElement('img');
+const mainText = document.createElement('p');
+
+function generateInformationAboutModel(event){
+    let buttonId = Number(event.target.id);
+    informationAboutModel.forEach((value) => {
+        if (value.id === buttonId){
+            mainInfoBlock.prepend(mainTitle);
+            mainTitle.classList.add('about-product__main-title');
+            mainTitle.innerHTML = `About BMW ${value.series}`;
+
+            mainInfoBlock.append(mainText);
+            mainText.classList.add('about-product__description');
+            mainText.innerHTML = value.information;
+
+            mainInfoBlock.append(mainImage);
+            mainImage.classList.add('about-product__main-image');
+            mainImage.src = value.images;
+        }
+    })
+}
+
+button.addEventListener('click', function (event) {
+    if (event.target.closest('button')){
```

Comment:
I think this was better `button.addEventListener('click', generateInformationAboutModel)`

Reply:
> Я думаю, що це було краще`button.addEventListener('click', generateInformationAboutModel)`

but I'm done with delegation? didn't need to do that?

Reply:
```javascript
const menuList = document.querySelector('.about-product__navigation');
buttons.addEventListener('click', generateInformationAboutModel);

```

Reply:
> ```js
> const menuList = document.querySelector('.about-product__navigation');
> buttons.addEventListener('click', generateInformationAboutModel);
> ```

Sorry me, but i don't understand you

Reply:
> ```js
> const menuList = document.querySelector('.about-product__navigation');
> buttons.addEventListener('click', generateInformationAboutModel);
> ```

I returned to the previous implementation method